keyboards/rad/keymaps/via/rules.mk

```diff
@@ -1 +1 @@
-VIA_ENABLE = yes
+	VIA_ENABLE = yes

```

Comment:
```suggestion
VIA_ENABLE = yes
```

Reply:
You added the tabs, I did not tell you to do that.

> This file is using Windows line endings, it should be Unix.

Reply:
You've going to have to fix this locally. GitHub keeps using Windows line endings.